libs/newsletters-data-client/src/lib/newsletter-data-type.ts

```diff
@@ -72,7 +72,7 @@ export const thrasherOptionsSchema = z.object({
 export type ThrasherOptions = z.infer<typeof thrasherOptionsSchema>;
 
 export const newsletterCategoriesSchema = z
-	.enum(['article-based', 'fronts-based', 'manual-send', 'other'])
+	.enum(['', 'article-based', 'fronts-based', 'manual-send', 'other'])
```

Comment:
We do this for other fields already so not a thing for this PR, but it's not ideal to be adding the empty string to the enum since its part of the schema for the actual `NewsletterData` type - I.E. this makes an empty string a valid value for `NewsletterData.category` (but the validation on the step will prevent it actually being selected). 

I'll put in a health ticket to look at this point. 🤔

Reply:
That's a very good point - I was making it consistent with existing steps (and I think I'd done it that way originally as a workaround for it being a mandatory field), but if there's a better way to tackle it that's definitely worth looking at separately